server/lib/init_log.rb

```diff
@@ -22,6 +22,17 @@ def self.log_change_deployment(deployment = nil)
     end
   end
 
+  def self.start_collect_satellite_logs(deployment)
+    self.log
+
+    FileUtils.mkdir_p(self.log_file_dir(deployment)) unless File.exist?(self.log_file_dir(deployment))
+    @log.collect(self.log_file_dir(deployment))
+  end
+
+  def self.stop_collect_satellite_logs
+    @log.stop_collect
```

Comment:
I think it would be better to call `log.stop_collect` instead of `@log.stop_collect` here (as calling this method without having called `log` first could raise an undefined method on nil error). In fact, I'd probably refactor this class to use `log` everywhere you're calling `@log` and drop the `self.log` calls you have at the beginning of methods.